torch_harmonics/convolution.py

```diff
@@ -258,7 +258,7 @@ def __init__(
         self,
         in_channels: int,
         out_channels: int,
-        kernel_shape: Union[int, List[int]],
+        kernel_shape: Union[int, Tuple[int]],
```

Comment:
This should probably be `Union[int, Tuple[int], Tuple[int, int]]`

Reply:
fixed